tests/app/main/views/test_suppliers.py

```diff
@@ -33,6 +51,24 @@ def test_services_link_is_shown_to_users_with_right_roles(self, data_api_client,
             "Role {} {} see the link".format(role, "can not" if link_should_be_visible else "can")
         )
 
+    @pytest.mark.parametrize("role, link_should_be_visible", [
+        ("admin", False),
```

Comment:
I just double-checked the permissions document and `admin` should definitely be able to "Update a supplier’s company name"

Reply:
OK - will add that back in.